packages/services/f-feature-management/test/endToEnd.test.js

```diff
@@ -0,0 +1,20 @@
+import featureManagement from '../src/index';
+import expectations from './data/e2e-expectations.json';
+
+const { getBooleanValue, getIntegerValue, getStringValue } = featureManagement('je-web-core');
+
+const functionMap = { boolean: getBooleanValue, integer: getIntegerValue, string: getStringValue };
+
+describe.skip('End-to-End Tests', () => {
+    expectations.expectations.forEach(ex => {
+        describe(ex.description, () => {
+            const { value: expectedValue, experimentKey: expectedKey, experimentVariant: expectedVariant } = ex.expectation;
+            const feature = functionMap[ex.valueType](ex.feature);
+            expect(feature.value).toBe(expectedValue);
+            if (expectedKey && expectedVariant) {
+                expect(feature.experimentKey).toBe(expectedKey);
+                expect(feature.experimentVariant).toBe(expectedVariant);
+            }
+        });
+    });
```

Comment:
There's a way of doing this idiomatically to jest, and it looks something like:
```suggestion
    describe.each(expectations)('Expectation %# - ', ({ description, expectation, feature, valueType }) => {
        it(description, () => {
            const { value: expectedValue, experimentKey: expectedKey, experimentVariant: expectedVariant } = expectation;
            const feature = functionMap[valueType](feature);
            expect(feature.value).toBe(expectedValue);
            if (expectedKey && expectedVariant) {
                expect(feature.experimentKey).toBe(expectedKey);
                expect(feature.experimentVariant).toBe(expectedVariant);
            }
        });
    });
```

Reply:
Thanks, this is much cleaner.